iceoryx_examples/ice_multi_publisher/README.md

```diff
@@ -0,0 +1,262 @@
+
+    std::thread sender1(send, 1, "Instance", std::chrono::milliseconds(500));
+    std::thread sender2(send, 2, "Instance", std::chrono::milliseconds(1000));
+
+Once Ctrl+C is pressed, we leave the publisher loops and join the threads.
+
+    sender1.join();
+    sender2.join();
+
+### Subscriber application
+
+We create a subscriber via
+    iox::popo::TypedSubscriber<CounterTopic> subscriber({"Group", "Instance", "Counter"});
+
+and immediately subscribe.
+    subscriber.subscribe();
+
+Notice that all identifiers match the ones provided by the two publishers.
+
+We periodically wake up
+
+    std::this_thread::sleep_for(std::chrono::seconds(1));
+    while (subscriber.hasNewSamples())
+
+When there are new samples we display them on the console.
+
+    subscriber.take()
+        .and_then([](iox::popo::Sample<const CounterTopic>& sample) {
+            std::cout << "Received: " << *sample.get() << std::endl;
+        })
+
+The topic is displayed by providing a defintion of ``operator<<`` which prints the id and counter to the console.
+The displayed counters are monotonically increasing for each id but between different publishers the order of data arrival is indeterminate due to concurrent sending.
+
+We also handle potential errors
+
+    .or_else([](iox::popo::ChunkReceiveError) { std::cout << "Error while receiving." << std::endl; });
+
+and wait for some time before looking for data again.
+
+    std::cout << "Waiting for data ... " << std::endl;
+
+When Ctrl+C is pressed we exit the loop and unsubscribe
+
+    subscriber.unsubscribe();
+
+before joining the receiver thread
+
+    receiver.join();
+
+### Resubscriber application
+
+We first define a bound for the maximum number of samples the subscriber can hold.
+
+    constexpr uint64_t MAX_NUMBER_OF_SAMPLES{4U};
+
+We also define some time period for which we will later unsubscribe.
+
+    constexpr uint64_t UNSUBSCRIBED_TIME_SECONDS{3U};
+
+We create a subscriber via
+    iox::popo::TypedSubscriber<CounterTopic> subscriber({"Group", "Instance", "Counter"});
+
+and immediately subscribe.
+    subscriber.subscribe();
+
+The topic we subscribe to is the same as for the subscriber application.
+
+We periodically unsubscribe for a specified time.
+
+    subscriber.unsubscribe();
+    std::cout << "Unsubscribed ... Subscribe in " << UNSUBSCRIBED_TIME_SECONDS << " seconds" << std::endl;
+    std::this_thread::sleep_for(std::chrono::seconds(UNSUBSCRIBED_TIME_SECONDS));
+
+We resubscribe and specify the maximum number of samples the subscriber is able to hold ``maxNumSamples``. 
+
+    maxNumSamples = maxNumSamples % MAX_NUMBER_OF_SAMPLES + 1U; 
+    subscriber.subscribe(maxNumSamples);
+
+This number changes cyclically from 1 up to 4. This means that in some cases we will only receive one sample, even when more were send by the two publishers.
```

Comment:
Please mention here that the others or dropped as the queue is overflowing and old samples are replaced by new ones

Reply:
As you commented below, this is already done.